README.md

```diff
@@ -128,9 +135,14 @@ You may run the IBM Z Accelerated for Snap ML container image on IBM Linux on Z
 or IBM® z/OS® Container Extensions (IBM zCX).
 
 _Note. The IBM Z Accelerated for Snap ML will transparently target the IBM
-Integrated Accelerator for AI on IBM® z16™ and later. However, if using the
-IBM Z Accelerated for Snap ML on either an IBM z14™ or an IBM z15™, Snap ML
-will transparently target the CPU with no changes to the model._
+Integrated Accelerator for AI on IBM z16 and later. However, if using the IBM Z
+Accelerated for Snap ML on either an IBM z15 or an IBM z14, Snap ML will
+transparently target the CPU with no changes to the model._
+
+# Technical Support <a id="contact"></a>
+
+Information regarding technical support can be found
+[here](https://github.com/IBM/ai-toolkit-for-z-and-linuxone).
 
 # Versions and Release cadence <a id="versions"></a>
```

Comment:
Would suggest changing this to `Versioning Policy and Release Cadence` to be consistent with above.